pkg/project/registry/projectrequest/delegated/delegated.go

```diff
@@ -201,14 +201,8 @@ func (r *REST) waitForRoleBinding(namespace, name string) {
 	backoff := retry.DefaultBackoff
 	backoff.Steps = 6 // this effectively waits for 6-ish seconds
 	err := wait.ExponentialBackoff(backoff, func() (bool, error) {
-		roleBindings, _ := r.roleBindings.RoleBindings(namespace).List(labels.Everything())
-		for _, roleBinding := range roleBindings {
-			if roleBinding.Name == name {
-				return true, nil
-			}
-		}
-
-		return false, nil
+		_, err := r.roleBindings.RoleBindings(namespace).Get(name)
+		return (err == nil), nil
```

Comment:
Why the redundant parentheses?

@liggitt there wasn't some archaic reason this was using list before right?

Reply:
maybe the lister we implemented for the virtual resources didn't implement Get?